join/join-testnet.sh

```diff
@@ -37,6 +42,11 @@ for arg in "$@"; do
   esac
 done
 
+if $STOP_ME; then
+  echo "Please checkout the correct version tag. See README for instructions."
```

Comment:
For the next release, we would have to update this logic, otherwise it would stop on a release branch too.